spec/javascript/testUtils/wrapperUtil.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import { shallow, mount } from 'enzyme'
+import Context from '~/components/supplemental_application/context'
+import { Form } from 'react-final-form'
+
+const formNode = (application, formToChildrenFunc) => (
+  <Form
+    onSubmit={() => null}
```

Comment:
should we make this a mock? or be able to pass a mock? I suppose we can add when we find that it's needed

Reply:
I think theres a lot that can and should be added but I figure we can add to these utils as we need new features.